Makefile

```diff
@@ -41,13 +42,15 @@ release: .prebuild
 
 google_release: .prebuild
 	@xcodebuild \
-		TEAM_ID=EQHXZ8M8AV \
+		TEAM_ID=${TEAM_ID} \
+		DEVELOPMENT_TEAM=${TEAM_ID} \
```

Comment:
Move these changes to the release rule - the intention for this was that release: is used for development and/or non-Google builds and google_release is for us when making a signed release.

We're planning to remove the google_release rule soon.